flyteadmin/auth/cookie.go

```diff
@@ -112,25 +113,29 @@ func ReadSecureCookie(ctx context.Context, cookie http.Cookie, hashKey, blockKey
 	return "", errors.Wrapf(ErrSecureCookie, err, "Error reading secure cookie %s", cookie.Name)
 }
 
-func NewCsrfToken(seed int64) string {
```

Comment:
Removed support for seeding since its apparently unwise to seed the token based on wall clock time, as that makes it potentially predictable for the user.